src/formatDate.js

```diff
@@ -50,7 +50,38 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateInput = date;
+  const oldFormat = fromFormat;
+  const newFormat = toFormat;
```

Comment:
What's the purpose of creating these variables? You can just use parameters directly